cli/tests/testdata/bench/no_check.out

```diff
@@ -1,5 +1,9 @@
-error: Uncaught TypeError: Cannot read properties of undefined (reading 'fn')
+error: TypeError: Cannot read properties of undefined (reading 'fn')
 Deno.bench();
      ^
     at [WILDCARD]
     at [WILDCARD]/bench/no_check.ts:1:6
+This error was not caught from a benchmark and caused the bench runner to fail on the referenced module.
```

Comment:
Makes the benchmark and test runner more consistent

Reply:
Very nice 👍